resources/views/auth/application/status_update_component.blade.php

```diff
@@ -1,19 +1,33 @@
 <div class="right">
-    @foreach (\App\Models\ApplicationForm::STATUSES as $st)
-    @if($st != \App\Models\ApplicationForm::STATUS_IN_PROGRESS || user()->isAdmin())
+    <div>
```

Comment:
also overridden by https://github.com/EotvosCollegium/mars/pull/574.
What was the point here anyways?

Reply:
For some reason, it did not update the status for me and I rewrote the whole thing :sweat_smile: